lib/sentry.ex

```diff
@@ -229,13 +207,7 @@ defmodule Sentry do
     end
   end
 
-  @spec unix_timestamp :: Number.t
-  defp unix_timestamp do
-    {mega, sec, _micro} = :os.timestamp()
-    mega * (1000000 + sec)
-  end
-
-  @spec unix_timestamp :: String.t
+  @spec iso8601_timestamp :: String.t
   defp iso8601_timestamp do
```

Comment:
Use the elixir DateTime methods http://elixir-lang.org/docs/stable/elixir/DateTime.html